src/main/java/org/alliancegenome/curation_api/services/associations/constructAssociations/ConstructGenomicEntityAssociationService.java

```diff
@@ -59,9 +59,11 @@ public ObjectResponse<ConstructGenomicEntityAssociation> upsert(ConstructGenomic
 			return null;
 		}
 		dbEntity = constructGenomicEntityAssociationDAO.persist(dbEntity);
+/*
```

Comment:
Don't we need this code?

Reply:
It does not needed to be done as this is only adding the association to the construct object but the association already knows about it. It would only be needed if you want to see it on the construct object which we don;t in this load.